### ISSUE TITLE
Cryptococcus NTR: Proliferation/survival in macrophages

### ISSUE THREAD

**Issue by dianeoinglis** [microbialphenotypes/OMP-ontology#179]
Hi,

I can't find the ticket for the NTR for "OMP:0005111 ! resistance to killing by host cells phenotype" but I did not intend for this term to be requested in the same format as the APO because the term presumes sensitivity is the normal state.

The phenotype observed is the result of counting number of microbial cells (by CFU) within host cells, specifically macrophages, but different species of fungal pathogens have different abilities to survive and proliferate within macrophages. Some fungi are efficiently killed by macrophages but this is not true for the intracellular fungal pathogens that can readily survive and proliferate within macrophages. 

Can "OMP:0005111 ! resistance to killing by host cells phenotype" and the related terms be obsoleted and replaced with a less presumptuous term?

Any of the following terms would be reasonable descriptions for the phenotype and they avoid the interpretation of the result as the APO was constructed: 
"survival in macrophages"
"proliferation in macrophages" 
"growth within macrophages"
"colonization of macrophages"

Thank you!
Diane

**Comment by dsiegele** [comment]
Diane, Should 'survival in microphages' and 'proliferation in macrophages' be two separate phenotypes?  Debby

**Comment by dsiegele** [comment]
Diane,

OMP:0005111 is supposed to be an umbrella term that covers both resistance and sensitivity. I rewrote the definition to state this.  New def: A microbe-host interaction phenotype relating to the resistance or sensitivity of a microbe to killing by a host.

Below is the organization I'm thinking of for this node. Please send feedback when you have a chance.

Cheers,

Debby 

OMP:0005111 resistance to killing by host phenotype (is_a microbe-host interaction phenotype)
.....OMP:0005139 presence of resistance to killing by host
..........OMP:0007452 presence of survival in macrophages
.....OMP:0005140 absence of resistance to killing by host
..........OMP:0007453 absence of survival in macrophages
.....OMP:0005141 altered resistance to killing by host
..........OMP:0005143 decreased resistance to killing by host
...............OMP:0007404 abolished resistance to killing by host
..........OMP:0005142 increased resistance to killing by host
..........OMP:0007454 altered survival in macrophages

.....OMP:0007455 survival in macrophages phenotype
..........OMP:0007452 presence of survival in macrophages
..........OMP:0007453 absence of survival in macrophages
..........OMP:0007454 altered survival in macrophages
...............this term would then have the children increased, decreased, abolished survival in macrophages

Would the phenotype 'presence of proliferation in macrophages' be a child of 'presence of survival in macrophages' or a sibling? I think it would be a child term, since proliferation requires survival. What do you think?

**Comment by dsiegele** [comment]
still need to make terms for proliferation in macrophages

**Comment by dianeoinglis** [comment]
Yes. I agree that survival and proliferation are distinct phenotypes.

Diane
